ipldpinner/set.go

```diff
@@ -219,13 +219,15 @@ func walkItems(ctx context.Context, dag ipld.DAGService, n *merkledag.ProtoNode,
 	// readHdr guarantees fanout is a safe value
 	fanout := hdr.GetFanout()
 	for i, l := range n.Links()[fanout:] {
-		if err := fn(i, l); err != nil {
```

Comment:
I thought shadowing `err` was generally something to avoid.  The primary reason being that someone may write a deferred function that closes over `err` and checks its value, and the author may not notice the shadowing.